Gemfile

```diff
@@ -28,26 +28,28 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+gem 'pg', '~> 1.2.3'
```

Comment:
Please add the rubocop gem to your project: https://github.com/rubocop/rubocop-rails

The rubocop gem acts as a Linter for rails projects - the default configuration should be good